OpenRA.Mods.Common/Activities/Demolish.cs

```diff
@@ -63,8 +63,7 @@ protected override void OnInside(Actor self)
 					building.Lock();
 
 				for (var f = 0; f < flashes; f++)
-					w.Add(new DelayedAction(flashesDelay + f * flashInterval, () =>
-						w.Add(new FlashTarget(target, ticks: flashDuration))));
+					w.Add(new FlashTarget(target, ticks: flashDuration, delay: flashesDelay + f * flashInterval));
```

Comment:
I just carried over that approach from the old lines, no specific reason other than that (I guess that was originally done to avoid the explicit `null` for `asPlayer`).

I've already done that locally, but it doesn't fix the regression @abcdefg30 pointed out.